### PR TITLE
Run Windows tests without Linux

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

In order to really test Windows agents we need to ensure that they are hit by tests we are running. To do that we need to remove private Linux nodes and run tests only on Windows node.

We can't remove Linux public nodes since this results in tests not starting at all.

This PR fixes dcos-diagnostics tests to accept clusters without Linux private nodes and to use Docker App that's running on Windows and Linux.

This PR skip couple of metrics tests since statsd metrics are not working on Windows and apps used in tests do not run on Windows.

## Corresponding DC/OS tickets (required)

Fises: 
  - [D2IQ-65996](https://jira.d2iq.com/browse/D2IQ-65996)

## Related tickets (optional)

  - [D2IQ-66051](https://jira.mesosphere.com/browse/D2IQ-66051)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -262,19 +263,24 @@ def get_nodes_from_response(response) -> List[str]:
             set(master_nodes).symmetric_difference(set(dcos_api_session.masters))
         )
 
-        agent_nodes_response = check_json(
-            dcos_api_session.health.get('/units/dcos-mesos-slave.service/nodes', node=master))
-        agent_nodes = get_nodes_from_response(agent_nodes_response)
+        linux_agent_nodes = list()
```
Comment (jeschkies) [inline packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: I think we have to do it like we do down in Windows

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required CI change

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary @KostiantynOleksiienko

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary @KostiantynOleksiienko

**Pull request merged**